docs/service-catalog/docs/020-architecture-service-catalog.md

```diff
@@ -2,16 +2,14 @@
 title: Architecture
 ---
 
-The diagram and steps describe the basic Service Catalog workflow:
+The diagram and steps describe the Service Catalog workflow and the roles of specific cluster and Environment-wide resources in this process:
 
-![Service Catalog flow](assets/service-catalog-flow.png)
+![Service Catalog flow](assets/service-catalog-flow.svg)
 
-1. During the Kyma installation, the system registers the default Service Brokers in the Kyma cluster. The cluster administrator can manually register other Service Brokers in the Kyma cluster.
+1. The Kyma installation results in the registration of the default Service Brokers in the Kyma cluster. The Kyma administrator can manually register other ClusterServiceBrokers in the Kyma cluster. The Kyma user can also register a Service Broker (ServiceBroker) in a given Environment.
 
-2. Each ClusterServiceBroker specifies ClusterServiceClasses and service variations called ClusterServicePlans that are available to Kyma users.
+2. Inside the cluster, each ClusterServiceBroker exposes services that are ClusterServiceClasses in their different variations called ClusterServicePlans. Similarly, the ServiceBroker registered in a given Environment exposes ServiceClasses and ServicePlans only in this specific Environment.
 
-3. The user gets a list of the available services in the Kyma web console or CLI.
+3. In the Console UI or CLI, the Kyma user lists all exposed cluster-wide and Environment-specific services and requests to create their instances in the Environment (ServiceInstances).
```

Comment:
`their instances in the Environment (ServiceInstances)` ?

Reply:
What's unclear about that sentence?